setup.py

```diff
@@ -9,5 +9,8 @@
     author_email='yuvipanda@gmail.com',
     license='3 Clause BSD',
     packages=find_packages(),
-    install_requires=['bcrypt']
+    install_requires=['bcrypt'],
+    package_data={
+        '': ['*.html'],
```

Comment:
I didn't know you could leave the key empty! I learnt something new today :)

Reply:
just followed the example that I saw [here](https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files) :)